tgui/packages/tgui/interfaces/CheckboxInput.tsx

```diff
@@ -51,9 +51,9 @@ export const CheckboxInput = (props) => {
 
     setSelections(newSelections);
   };
-  const selectionIndexes = selections.map(
-    (selected: string) => items.indexOf(selected) + 1,
-  );
+
+  const selectionsWithIndexes: (selections: string[], items: string[]) => [string, number][] =
+  (selections, items) => selections.map((selected) => [selected, items.indexOf(selected) + 1]);
```

Comment:
Prefer typing params directly rather than functions, it's much more straight forward. Typing an arrow function is only useful in rare cases

```suggestion
  const selectionsWithIndexes =
  (selections: string[], items: string[]): [string, number][] => selections.map((selected) => [selected, items.indexOf(selected) + 1]);
```